packages/common/src/search/searchContentEntities.ts

```diff
@@ -0,0 +1,105 @@
+import { ISearchOptions, searchItems } from "@esri/arcgis-rest-portal";
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-types";
+import {
+  expandContentFilter,
+  Filter,
+  getNextFunction,
+  IHubSearchOptions,
+  serializeContentFilterForPortal,
+} from ".";
+import { ISearchResponse } from "..";
+
```

Comment:
Would appreciate input from @tomwayson on this - it all works - even after this is merged

Reply:
I'm not sure what you are expecting not to work, but we definitely should **not** be importing from `"."` and `".."`  b/c of circular deps.

Reply:
was more the  use of `createContentEntitySearchFn` which holds a closure over a `ItemToEntityFunction<T>`. It's really useful but a little complex. That said, the `IHubItemEntity` interface now requires an `fromItem(IItem):Promise<T>` which is the function needed in the closure... so... complex, but not random?